edvart/report_sections/dataset_overview.py

```diff
@@ -449,7 +449,7 @@ def data_types(df: pd.DataFrame, columns: Optional[List[str]] = None) -> None:
     """
     if columns is not None:
         df = df[columns]
-    dtypes = df.apply(
+    dtypes = df.apply(  # type: ignore
```

Comment:
Why ignore? Isn't it `pd.Series`? I.e. `dtypes: pd.Series = df.apply(...`

Reply:
Added a comment. The ignore is not because of the result type, but because of the parameters of the `apply` function.